<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>InsertDiag</class>
 <widget class="QDialog" name="InsertDiag">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>480</width>
    <height>240</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QDialog#InsertDiag 
{
	background : gray
}

QLabel
{
	color: white
}</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <item row="0" column="0">
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="2">
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QLineEdit" name="entry1"/>
       </item>
       <item>
        <widget class="QLineEdit" name="entry2"/>
       </item>
       <item>
        <widget class="QLineEdit" name="entry3"/>
       </item>
       <item>
        <widget class="QLineEdit" name="entry4">
         <property name="text">
          <string>Preenchido automaticamente se Azimute de Referência for fornecido</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="entry5"/>
       </item>
       <item>
        <widget class="QLineEdit" name="entry6"/>
       </item>
      </layout>
     </item>
     <item row="0" column="0">
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QLabel" name="label1">
         <property name="text">
          <string>Estação (Obrigatório)</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label2">
         <property name="text">
          <string>Ponto Visado</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label5">
         <property name="text">
          <string>Ângulo Horizontal</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label3">
         <property name="text">
          <string>Azimute</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label4">
         <property name="text">
          <string>Distância Horizontal</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label6">
         <property name="text">
          <string>Observações</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item row="0" column="1">
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeType">
        <enum>QSizePolicy::Preferred</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>10</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item row="1" column="0">
    <widget class="QDialogButtonBox" name="diag_ok">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>diag_ok</sender>
   <signal>accepted()</signal>
   <receiver>InsertDiag</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>diag_ok</sender>
   <signal>rejected()</signal>
   <receiver>InsertDiag</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
